packages/visx-grid/src/grids/GridAngle.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import cx from 'classnames';
+import Line, { LineProps } from '@visx/shape/lib/shapes/Line';
+import { Group } from '@visx/group';
+import { ScaleInput, getTicks } from '@visx/scale';
+import { Point } from '@visx/point';
+
+import { CommonGridProps, GridScale } from '../types';
+import polarToCartesian from '../utils/polarToCartesian';
+
+export type GridAngleProps<Scale extends GridScale> = CommonGridProps & {
+  /** `@visx/scale` or `d3-scale` object used to convert value to position. */
+  scale: Scale;
+  /**
+   * Exact values used to generate grid lines using `scale`.
+   * Overrides `numTicks` if specified.
+   */
+  tickValues?: ScaleInput<Scale>[];
+  /**
+   * Radius which determines the start position of polar lines
+   */
+  innerRadius?: number;
+  /**
+   * Radius which determines the end position of polar lines
+   */
+  outerRadius?: number;
+  /**
+   * The class name applied to all polar lines.
+   */
+  lineClassName?: string;
+};
+
+export type AllGridAngleProps<Scale extends GridScale> = GridAngleProps<Scale> &
+  Omit<
+    LineProps & Omit<React.SVGProps<SVGLineElement>, keyof LineProps>,
+    keyof GridAngleProps<Scale>
+  >;
+
+export default function GridAngle<Scale extends GridScale>({
+  className,
+  innerRadius = 0,
+  left = 0,
+  lineClassName,
+  lineStyle,
+  numTicks = 10,
+  outerRadius = 0,
+  scale,
+  stroke = '#eaf0f6',
+  strokeDasharray,
+  strokeWidth = 1,
+  tickValues,
+  top = 0,
+  ...restProps
+}: AllGridAngleProps<Scale>) {
+  const ticks = tickValues ?? getTicks(scale, numTicks);
+  return (
+    <Group className={cx('visx-grid-angle', className)} top={top} left={left}>
+      {ticks.map((tick, i) => {
+        const angle = (scale(tick) as number) - Math.PI / 2;
```

Comment:
can we avoid this coercion? 

I believe `@visx/scale` exports a `coerceNumber` `util` that can help. I think the usage would be something like `(coerceNumber(scale(tick)) ?? Math.PI / 2) - Math.PI / 2` (`?? Math.PI / 2` would default the angle to `0`)

Reply:
Okay, will make the change